src/components/app-menu/container.tsx

```diff
@@ -12,7 +12,14 @@ export interface Properties {
 
 export class Container extends React.Component<Properties, {}> {
   static mapState(state: RootState): Partial<Properties> {
-    const { zns: { value: { route } }, apps: { selectedApp: { type } } } = state;
```

Comment:
I kind of like the single line destructuring version... but my eyes will readjust over time.